docs/9.0/guides/formulas/hyperformula-integration.md

```diff
@@ -122,6 +122,7 @@ This example is more typical of data grids than spreadsheets. Calculations are p
 ```js
```

Comment:
is this change to the `9.0` required?

Reply:
I suppose that 9.0 should be the same as on production (migration point that shows that we introduced and deprecated Formulas plugin). The `next` version continues the docs having only the new plugin in mind.

Reply:
AFAIK, we plan to remove 9.0 before release because we generated 9.0 only for version selection tests, isn't it?
Then, we generate new version based on the `next`

Reply:
I don't know what the plan is for 9.0. I would suppose that the 9.0 version would be served using the new engine.